packages/python-androguard/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-androguard
+pkgver=3.2.1
+pkgrel=1
+pkgdesc='Androguard is a full python tool to play with Android files.'
+arch=('any')
+url='https://pypi.org/project/androguard'
+license=('MIT')
+depends=('python' 
```

Comment:
They are needed. The `install_requires` are not in the python 3.4 indention block and the different scrip[ts wont work without them

Reply:
@ikstream You are right, I misread it, only the print is in the if 3.4 scope, that is the problem with language like python that use indentation for blocks. 
And for the test I apologize but i'm building this PKG only as a dependency of another so I don't really know how to test it.

Reply:
Was very likely me, as i was copying from file into Github.
How to test depends on the package that is added. androguard for example provides some
command line tools you can call.
If it's a python library, just try to import it in python repl and probably do a basic call, this will show most problems with dependencies.